syntax = "proto3";
package ProtocolMessage;

message ProtoMessage
{
	int32 proto_id = 1;
	bytes proto_data = 2;
}

